.github/workflows/publish.yml

```diff
@@ -16,16 +16,19 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     - run: git clone --single-branch --no-tags --depth 1 -b master https://grafanabot:${{ secrets.GH_BOT_ACCESS_TOKEN }}@github.com/grafana/website-sync ./.github/actions/website-sync
+    - name: Get yarn cache directory path
+      id: yarn-cache-dir-path
+      run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
     - uses: actions/cache@v2.1.6
       with:
-        path: '**/node_modules'
-        key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+        path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+        key: yarn-${{ hashFiles('**/yarn.lock') }}
     - name: generate-packages-docs
       uses: actions/setup-node@v2.4.0
       id: generate-docs
       with:
         node-version: '14'
-    - run: yarn install --pure-lockfile --no-progress
+    - run: YARN_ENABLE_PROGRESS_BARS=false yarn install --immutable
```

Comment:
I see `YARN_ENABLE_PROGRESS_BARS=false` a lot in this PR, Yarn sets that automatically when it detects its running in a CI https://yarnpkg.com/configuration/yarnrc#enableProgressBars so it's unnecessary here, unsure if `drone` is detected as a CI though